apps/users/serializers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .auth_serializers import UserRegistrationSerializer, LoginSerializer, LoginResponseSerializer, PassWordResetRequestSerializer, PasswordResetConfirmationSerializer
```

Comment:
We are normalizing full import statements.

```suggestion
from apps.authentication.auth_serializers import UserRegistrationSerializer, LoginSerializer, LoginResponseSerializer, PassWordResetRequestSerializer, PasswordResetConfirmationSerializer
```

So let's adopt the lines above and follow same for the rest.